frontend/testing/cypress/src/integration/studio/resourceadm.js

```diff
@@ -0,0 +1,51 @@
+/// <reference types="cypress" />
+/// <reference types="../../support" />
+
+import * as texts from "@altinn-studio/language/src/nb.json";
+
+// First Cypress tests of sub-repo Resourceadm: this is a work in progress
+
+context("Resourceadm", () => {
+  before(() => {
+    cy.studiologin(Cypress.env("autoTestUser"), Cypress.env("autoTestUserPwd"));
+    // the login session will last throughout the tests
```

Comment:
What about removing this comment? The code is very self-explained. :)
```suggestion
    cy.studiologin(Cypress.env("autoTestUser"), Cypress.env("autoTestUserPwd"));
```